### PR TITLE
AO3-5239 Fixes for single simple form style

### PR DESCRIPTION
**Pull request by sarken**
## Issue

https://otwarchive.atlassian.net/browse/AO3-5239

## Purpose

* Gets rid of weird red highlighting that appeared behind collection member autocomplete input when hovering over the empty space between autocomplete suggestions after you've already selected people from the autocomplete. 
* Makes the autocomplete look better when entering more than one token (and multiple lines of tokens).

## Testing Instructions

Refer to Jira.

### CODE REVIEW

File: public/stylesheets/site/2.0/07-interactions.css
Comment (redsummernight) [inline public/stylesheets/site/2.0/07-interactions.css:239]: `.autocomplete .dropdown ul li:hover, li.selected` is also present in public skins (Reversi, Snow, Snow Blue) and the skin wizard header styles, do those need to be updated as well?

Comment (sarken) [reply public/stylesheets/site/2.0/07-interactions.css:239]: `:sob_blood:` Yes, thanks, I'll do that once I finish Duolingo.

Comment (redsummernight) [reply public/stylesheets/site/2.0/07-interactions.css:239]: Looks good! Do you want to copy the public skins to staging (https://github.com/otwcode/otwarchive/pull/4247#issuecomment-1145315236) before merging?

And looks like we'll have to deal with the [Dreamwidth importing failures](https://otwarchive.atlassian.net/browse/AO3-6347) soon.

Comment (sarken) [reply public/stylesheets/site/2.0/07-interactions.css:239]: Yeah, I'll copy the skins over. Can you make an issue for disabling those tests?

Comment (sarken) [reply public/stylesheets/site/2.0/07-interactions.css:239]: Skins copied!

Comment (redsummernight) [reply public/stylesheets/site/2.0/07-interactions.css:239]: Filed [AO3-6353](https://otwarchive.atlassian.net/browse/AO3-6353).

Comment (redsummernight) [reply public/stylesheets/site/2.0/07-interactions.css:239]: Will merge after #4249.

**Review by redsummernight — APPROVED** [review]

**Pull request merged**
